vassal-app/src/main/java/VASSAL/build/module/GameRefresher.java

```diff
@@ -536,8 +537,11 @@ else if (!decksToDelete.isEmpty()) {
         log(Resources.getString("GameRefresher.refreshable_decks", refreshable));
         log(Resources.getString(options.contains("DeleteOldDecks") ? "GameRefresher.deletable_decks" : "GameRefresher.deletable_decks_2", deletable)); //NON-NLS
         log(Resources.getString(options.contains("AddNewDecks") ? "GameRefresher.addable_decks" : "GameRefresher.addable_decks_2", addable)); //NON-NLS
+
       }
     }
+    // After all refreshing, allow a custom finish...
+    GameModule.getGameModule().fireKeyStroke(NamedKeyStroke.of("VassalRefreshGHK"));
   }
```

Comment:
A simple, useful change. I have no problem with it being a pre-defined keystroke, nothing would be added by making it configurable.

Reply:
Refresher Doc updated.